common/src/sql/dbinit.sql

```diff
@@ -642,6 +642,53 @@ CREATE INDEX ON omicron.public.console_session (
     time_created
 );
 
+/*******************************************************************/
+
+/*
+ * IAM
+ */
+
+/*
+ * Users built into the system
+ *
+ * The ids and names for these users are well-known (i.e., they are used by
+ * Nexus directly, so changing these would potentially break compatibility).
+ */
+CREATE TABLE omicron.public.user_builtin (
+    /*
+     * Identity metadata
+     *
+     * TODO-cleanup This uses the "resource identity" pattern because we want a
+     * name and description, but it's not valid to support sort-deleting these
+     * records.
+     */
+    id UUID PRIMARY KEY,
+    name STRING(63) NOT NULL,
+    description STRING(512) NOT NULL,
+    time_created TIMESTAMPTZ NOT NULL,
+    time_modified TIMESTAMPTZ NOT NULL,
+    time_deleted TIMESTAMPTZ
+);
+
+CREATE UNIQUE INDEX ON omicron.public.user_builtin (name);
+
+/* User used by Nexus to create other users.  Do NOT add more users here! */
+INSERT INTO omicron.public.user_builtin (
+    id,
+    name,
+    description,
+    time_created,
+    time_modified
+) VALUES (
+    /* NOTE: this uuid and name are duplicated in nexus::authn. */
+    '001de000-05e4-0000-0000-000000000001',
```

Comment:
I think this UUID isn't strictly valid, though I don't think it's likely to bite us unless a UUID lib is trying to interpret the value

Reply:
Yeah, all of the built-in users have made-up uuids described in the comment in authn.rs.  I believe they all parse with `uuid::Uuid::from_str`, though.  When you say it's not strictly valid, did you mean that it's not consistent with any of the v1-v4 specs?

Reply:
yeah, and it's in the variant 0 (NCS) space, which has its own subfields

Reply:
Yeah, that's true.  How about I change all the built-in user uuids to v4 variant 1 (but otherwise keep their structure the same)?  Another option would be to stick with variant 0 (which appears obsolete) or maybe variant 2 (which appears irrelevant to us), but I don't think there's any particular reason to do that.  When I wrote up this scheme, I was definitely thinking of them as v4 ones.

Reply:
Done in 539e538e5c24a2cc894d10efdaba3be2f7e82ff4.